core/trino-main/src/test/java/io/trino/server/security/oauth2/TestingHydraService.java

```diff
@@ -97,21 +99,41 @@ public void start()
         return new FixedHostPortGenericContainer<>(HYDRA_IMAGE).withNetwork(network);
     }
 
-    public void createConsumer(String callback)
+    public void createClient(
+            String clientId,
+            String clientSecret,
+            TokenEndpointAuthMethod tokenEndpointAuthMethod,
+            String audience,
+            String callbackUrl)
     {
         createHydraContainer()
                 .withCommand("clients create " +
                         "--endpoint http://hydra:4445 " +
-                        "--id another-consumer " +
-                        "--secret consumer-secret " +
-                        "-g authorization_code,refresh_token " +
+                        "--id " + clientId + " " +
```

Comment:
Use the variant that passes arguments separately, rather than concatenating strings, which has issues with quoting / shell expansion
```java
.withCommand("clients", "create",
        "--endpoint", "http://hydra:4445",
        "--id", clientId,